### DISCUSSION TITLE
Problems with installing, seems to be related to doctrine/dbal

### DISCUSSION THREAD

**Discussion by leeovery** [laravel/nova-issues#3678] [Q&A] [answered]
- Laravel Version: ^8.12
- Nova Version: ~3.0
- PHP Version: 8.0.13

### Description:
Followed the instructions for installing via composer as per the docs but when trying to install nova I get the following error from composer:

```
# composer update --prefer-dist                                                         
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - laravel/nova[dev-master, v3.0.0, ..., 3.30.0] require doctrine/dbal ^2.9 -> found doctrine/dbal[v2.9.0, ..., 2.13.x-dev] but it conflicts with your root composer.json require (^3.0).
    - laravel/nova 3.x-dev is an alias of laravel/nova dev-master and thus requires it to be installed too.
    - Root composer.json requires laravel/nova ~3.0 -> satisfiable by laravel/nova[v3.0.0, ..., 3.x-dev (alias of dev-master)].
```

### Can be recreated:

1. Setup a fresh Laravel project using `laravel new testnova`
2. Add nova statis repo for installing via composer (as per docs)
3. Add `"laravel/nova": "~3.0"` to the composer.json file
4. Run `composer update --prefer-dist` again as per docs
5. All works fine to this point but...
6. Run `composer require doctrine/dbal` and the error will appear.

Requiring doctrine/dbal, as you know, is suggested in the Laravel docs to amend cols in migrations.

Thanks
Lee

**Comment by crynobone** [comment, marked as answer]
Use `composer require "doctrine/dbal:^2.9"`
